docs/adguard-browser-extension/mv3-version.md

```diff
@@ -0,0 +1,145 @@
+---
+title: AdGuard Browser Extension for Chrome MV3
+sidebar_position: 5
+---
+
+As MV2 extensions are being phased out in the Chrome Web Store, Chrome users may need to switch to MV3 extensions that are compatible with Chrome’s new API Manifest V3. Our thoughts on this topic are extensively discussed on [our blog](https://adguard.com/en/blog/tag/manifest-v3.html).
+
+Our MV3 extension effectively blocks ads and trackers while seamlessly managing social widgets, banners, and video ads. Most users will not notice any operational differences, but there are some limitations and changes to be aware of.
+
+## Where to find our extensions
+
+The **MV3 version** has replaced our old extension in the [Chrome Web Store](https://chromewebstore.google.com/detail/adguard-adblocker/bgnkhhnnamicmpeenaelnjfhikgbkllg).
```

Comment:
@AlinaNaletova so-called "[old extension](https://chromewebstore.google.com/detail/adguard-adblocker/bgnkhhnnamicmpeenaelnjfhikgbkllg)" has not been replaced now. so if it is merged now, it would be wrong, isn't it?

Reply:
fixed it